packages/sharec-core/lib/index.js

```diff
@@ -50,6 +50,7 @@ const { errorCauses, InternalError } = require('./errors')
  * @property {object} [local] Configs from target package
  * @property {object} [mergedConfigs] Processed configs from upcoming package
  * @property {object} [cache] Previously installed configuration
+ * @property {array} [binaries] Binary files
```

Comment:
Type should be `string[]` and parameter description should be much clean, like `Binaries files paths`